src/test/java/edu/cornell/kfs/coa/fixture/IndirectCostRecoveryAccountChangeFixture.java

```diff
@@ -0,0 +1,57 @@
+package edu.cornell.kfs.coa.fixture;
+
+import java.math.BigDecimal;
+
+import edu.cornell.kfs.coa.businessobject.IndirectCostRecoveryAccountChange;
+
+public enum IndirectCostRecoveryAccountChangeFixture {
+	ICR_CHANGE_1111111_98_PERCENT_ACTIVE("IT", "1111111", new BigDecimal(98),
+			true),
+
+	ICR_CHANGE_1111111_98_PERCENT_INACTIVE("IT", "1111111", new BigDecimal(98),
+			false),
+
+	ICR_CHANGE_1111111_100_PERCENT_ACTIVE("IT", "1111111", new BigDecimal(100),
+			true),
+
+	ICR_CHANGE_1111111_100_PERCENT_INACTIVE("IT", "1111111",
+			new BigDecimal(100), false),
+
+	ICR_CHANGE_2222222_2_PERCENT_ACTIVE("IT", "2222222", new BigDecimal(2),
+			true),
+
+	ICR_CHANGE_2222222_2_PERCENT_INACTIVE("IT", "2222222", new BigDecimal(2),
+			false),
+
+	ICR_CHANGE_3333333_100_PERCENT_ACTIVE("IT", "3333333", new BigDecimal(100),
+			true),
+
+	ICR_CHANGE_3333333_100_PERCENT_INACTIVE("IT", "3333333",
+			new BigDecimal(100), false);
+
+	public String indirectCostRecoveryFinCoaCode;
+	public String indirectCostRecoveryAccountNumber;
+	public BigDecimal accountLinePercent;
+	public boolean active;
```

Comment:
It looks like only some of the fixtures were updated to have final variables. There are a few (including this one) that should be declaring these as "final" to prevent accidental updates after construction.